### ISSUE TITLE
Django RQ 0.9.1 RQ_SHOW_ADMIN_LINK option breaks new Pootle installs

### ISSUE THREAD

**Issue by theflow** [translate/pootle#4737]
I'm trying to follow the installation docs and I can't get it to work. I tried it both on my Mac and on a Ubuntu 12.04 machine. It always fails like this:

```pytb
(env) pootle@pootle0:~/pootle$ pootle --version
Traceback (most recent call last):
  File "/podio/pootle/pootle/env/bin/pootle", line 7, in <module>
    from pootle.runner import main
  File "/podio/pootle/pootle/env/local/lib/python2.7/site-packages/pootle/runner.py", line 19, in <module>
    from .core.utils.redis_rq import rq_workers_are_running
  File "/podio/pootle/pootle/env/local/lib/python2.7/site-packages/pootle/core/utils/redis_rq.py", line 12, in <module>
    from django_rq.queues import get_queue
  File "/podio/pootle/pootle/env/local/lib/python2.7/site-packages/django_rq/__init__.py", line 5, in <module>
    from .workers import get_worker
  File "/podio/pootle/pootle/env/local/lib/python2.7/site-packages/django_rq/workers.py", line 5, in <module>
    from .settings import EXCEPTION_HANDLERS
  File "/podio/pootle/pootle/env/local/lib/python2.7/site-packages/django_rq/settings.py", line 8, in <module>
    SHOW_ADMIN_LINK = getattr(settings, 'RQ_SHOW_ADMIN_LINK', False)
  File "/podio/pootle/pootle/env/local/lib/python2.7/site-packages/django/conf/__init__.py", line 46, in __getattr__
    self._setup(name)
  File "/podio/pootle/pootle/env/local/lib/python2.7/site-packages/django/conf/__init__.py", line 40, in _setup
    % (desc, ENVIRONMENT_VARIABLE))
django.core.exceptions.ImproperlyConfigured: Requested setting RQ_SHOW_ADMIN_LINK, but settings are not configured. You must either define the environment variable DJANGO_SETTINGS_MODULE or call settings.configure() before accessing settings.
```

If I set the settings, I get another error:

```pytb
(env) pootle@pootle0:~/pootle$ DJANGO_SETTINGS_MODULE="pootle.settings" pootle --version
Traceback (most recent call last):
  File "/podio/pootle/pootle/env/bin/pootle", line 7, in <module>
    from pootle.runner import main
  File "/podio/pootle/pootle/env/local/lib/python2.7/site-packages/pootle/runner.py", line 19, in <module>
    from .core.utils.redis_rq import rq_workers_are_running
  File "/podio/pootle/pootle/env/local/lib/python2.7/site-packages/pootle/core/utils/redis_rq.py", line 12, in <module>
    from django_rq.queues import get_queue
  File "/podio/pootle/pootle/env/local/lib/python2.7/site-packages/django_rq/__init__.py", line 5, in <module>
    from .workers import get_worker
  File "/podio/pootle/pootle/env/local/lib/python2.7/site-packages/django_rq/workers.py", line 5, in <module>
    from .settings import EXCEPTION_HANDLERS
  File "/podio/pootle/pootle/env/local/lib/python2.7/site-packages/django_rq/settings.py", line 8, in <module>
    SHOW_ADMIN_LINK = getattr(settings, 'RQ_SHOW_ADMIN_LINK', False)
  File "/podio/pootle/pootle/env/local/lib/python2.7/site-packages/django/conf/__init__.py", line 46, in __getattr__
    self._setup(name)
  File "/podio/pootle/pootle/env/local/lib/python2.7/site-packages/django/conf/__init__.py", line 42, in _setup
    self._wrapped = Settings(settings_module)
  File "/podio/pootle/pootle/env/local/lib/python2.7/site-packages/django/conf/__init__.py", line 115, in __init__
    raise ImproperlyConfigured("The SECRET_KEY setting must not be empty.")
django.core.exceptions.ImproperlyConfigured: The SECRET_KEY setting must not be empty.
```

Something seems to be fundamentally wrong with my setup, but I have no idea where to start looking.

I tried with Ubuntu (python 2.7.3, pip 8.1.2) and my Mac (python 2.7.8, pip 6.1.1). Both times installing Pootle 2.7.3. Here is a pip freeze

```
(env) pootle@pootle0:~$ pip freeze
click==6.6
cssmin==0.2.0
diff-match-patch==20121119
Django==1.7.11
django-allauth==0.25.2
django-assets==0.11
django-contact-form==1.0
django-overextends==0.4.1
django-redis==4.4.3
django-rq==0.9.1
django-transaction-hooks==0.2
elasticsearch==1.6.0
lxml==3.6.0
oauthlib==1.1.1
Pootle==2.7.3
python-openid==2.2.5
redis==2.10.5
requests==2.10.0
requests-oauthlib==0.6.1
rq==0.6.0
six==1.10.0
sphinx-me==0.3
translate-toolkit==1.13.0
urllib3==1.15.1
webassets==0.11
```

Any help is appreciated.

**Comment by dwaynebailey** [comment]
@theflow quick fix for now, while we try find a solution is the following `pip install django-rq==0.9.0`

**Comment by dwaynebailey** [comment]
This is the baby https://github.com/ui/django-rq/pull/20 that brings in the bustage into Pootle 2.7.3

**Comment by phlax** [comment]
@theflow @dwaynebailey there is a fix for this moving the rq check import in pootle.runner

**Comment by phlax** [comment]
@theflow thanks for reporting this - we will release 2.7.4 shortly with a fix for this issue

**Comment by theflow** [comment]
thank you for the quick response!

**Comment by julen** [comment]
@dwaynebailey I'm curious why you think the linked PR causes this? It's code that has been available since 2013 and django-rq version 0.6.

**Comment by phlax** [comment]
@julen i believe that the problem with the newer version of django-rq is that it imports earlier from django - ie before django.settings has been configured in pootle.runner

the fix i had in mind for 2.7.4 was just to pin the django-rq version, but for master i was thinking this was a better fix.

**Comment by dwaynebailey** [comment]
@julen mmmm you're right, I just quickly looked for first occurrence of that setting.

**Comment by unho** [comment]
A new Pootle 2.7.4 has been just released. Please let us know if the issue is now fixed for Pootle 2.7.
